### PR TITLE
add support for get and put manifest schema v1

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: docker_registry_client/manifest.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# code extracted from https://git.io/vM0EB used under license:
+#
+# Copyright (c) 2015 David Halls <https://github.com/davedoesdev/>
```
Comment (graingert) [inline docker_registry_client/manifest.py]: @davedoesdev I'd like to make this a lot smaller, so with your permission can I change this to be just one line.

```python
# code extracted from https://git.io/vM0EB used under license (MIT).
```

**Comment by graingert** [comment]
With this the return values of "get_manifest" are opaque cookies and should not be used by calling code, except to call put_manifest.

File: docker_registry_client/manifest.py (hunk 1 above)
Comment (davedoesdev) [inline docker_registry_client/manifest.py]: Yes, no problem

**Comment by graingert** [comment]
Please remember to do the changelog!

**Comment by munk** [comment]
Issue 35 is open to update the changelog and bump the version.

**Pull request merged**